### PR TITLE
testing

### PR DESCRIPTION
**Pull request by MichielVieijra**
Fout show() functie opgelost.

### CODE REVIEW

File: src/vlinder_toolkit/stations.py [hunk 1]
```diff
@@ -110,25 +110,28 @@ def __init__(self, station_name, network_name):
...
+            print('Network: ', self.network)
+            print('Call name: ', self.call_name)
+            print('Location: ', self.location)
+            print('latitude: ', self.lat)
+            print('longtitude: ',self.lon)
+           
+            print(' ------- Physiography info -------')
+            print('LCZ: ', self.lcz)
+            print(' ')
+            print(' ------- Observations info -------')
```
Comment (vergauwenthomas) [inline src/vlinder_toolkit/stations.py:132]: Move this out of the else loop. So the name and network can be printed. All the other attributes are None by default.

**Review by vergauwenthomas — CHANGES_REQUESTED** [review]
my review:

File: src/vlinder_toolkit/stations.py (hunk 1 above)
Comment (vergauwenthomas) [reply src/vlinder_toolkit/stations.py:132]: + remove the `print('hoi') `

**Comment by vergauwenthomas** [comment]
Make link to the issue number, and i will approve

**Comment by MichielVieijra** [comment]
https://github.com/vergauwenthomas/vlinder_toolkit/issues/22 solved by this code.

**Pull request merged**